src/System.Management.Automation/engine/PseudoParameters.cs

```diff
@@ -236,6 +236,6 @@ public string HelpFile
         /// </summary>
         public object Data { get; set; }
 
-        internal static readonly RuntimeDefinedParameter[] EmptyParameterArray = new RuntimeDefinedParameter[0];
+        internal static readonly RuntimeDefinedParameter[] EmptyParameterArray = Array.Empty<RuntimeDefinedParameter>();
```

Comment:
Perhaps we can inline `EmptyParameterArray `?

Reply:
Not in the PR.